.gitignore

```diff
@@ -26,7 +26,7 @@ docs/_build/
 nosetests.xml
 __pycache__/
 htmlcov/
-_dynamic/
+_dump/
```

Comment:
Rename workdir from "_dynamic" to "_dump".

Because a long time ago I did not understand what "_static" was meant to be...